app/Models/Book.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Models;
+
+use App\Models\Like as ModelsLikes;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Books extends Model
```

Comment:
Tên Model đặt số ít nhé
```suggestion
class Book extends Model
```